build/common/installer/scripts/tomlparser.rb

```diff
@@ -256,7 +262,7 @@ def get_command_windows(env_variable_name, env_variable_value)
     file.write(commands)
     commands = get_command_windows('AZMON_CLUSTER_COLLECT_ALL_KUBE_EVENTS', @collectAllKubeEvents)
     file.write(commands)
-    commands = get_command_windows('AZMON_CONTAINER_LOGS_EFFECTIVE_ROUTE', @containerLogsRoute)
+    commands = get_command_windows('AZMON_CONTAINER_LOGS_ROUTE', @containerLogsRoute)
```

Comment:
Is this teste for ADX route ?

Reply:
This just renaming the variable from AZMON_CONTAINER_LOGS_EFFECTIVE_ROUTE to AZMON_CONTAINER_LOGS_ROUTE and shouldnt impact ADX route but I can to test it  out ( or cover as part of bug bash) to ensure nothing impacts. can you please point me the instructions how to validate ADX route?

Reply:
verified adx route and looks good to me.